### PR TITLE
Release notes v1.1.1

### PR DESCRIPTION
**Pull request by kozer**
Related to pfHvTO-c6-p2

## Proposed Changes

Updated release notes for v1.1.1

## Testing Instructions

N/A

## Pre-merge Checklist

- [ ] Have you checked for TypeScript, React or other console errors?

### CODE REVIEW

File: RELEASE-NOTES.txt [hunk 1]
```diff
@@ -1,3 +1,9 @@
+1.1.1
+=====
+* Updated wp-now to fix missing curl and webp issues
+* Improved translation support and updated translations for multiple languages
+* Resolved server stop issue by adjusting force-stop functionality.
```
Comment (katinthehatsite) [inline RELEASE-NOTES.txt]: should we add a mention of the language switcher that we introduced?

Comment (kozer) [reply RELEASE-NOTES.txt]: Added one more item. 
What do you think @katinthehatsite

Comment (katinthehatsite) [reply RELEASE-NOTES.txt]: We could also potentially include: https://github.com/Automattic/studio/pull/495
As well as mention this improvement: https://github.com/Automattic/studio/issues/132

Comment (kozer) [reply RELEASE-NOTES.txt]: Isn't that too minor?

Comment (katinthehatsite) [reply RELEASE-NOTES.txt]: I have only done a release once so let's maybe ask someone more experienced - @wojtekn what do you think?

Comment (wojtekn) [reply RELEASE-NOTES.txt]: Agreed, it makes sense to add a language switcher. We can add an item saying we improved compatibility with WooCommerce, too.

File: RELEASE-NOTES.txt [hunk 2]
```diff
@@ -1,3 +1,10 @@
+1.1.1
+=====
+* Updated wp-now to fix missing curl and webp issues
```
Comment (wojtekn) [inline RELEASE-NOTES.txt]: Could you add PR links where possible, like we did for previous versions?

File: RELEASE-NOTES.txt (hunk 1 above)
Comment (wojtekn) [reply RELEASE-NOTES.txt]: Added in https://github.com/Automattic/studio/pull/523/commits/5bfc33f9dcd71909e0f514eb2dd6df6823c6379a

File: RELEASE-NOTES.txt (hunk 2 above)
Comment (wojtekn) [reply RELEASE-NOTES.txt]: Added in https://github.com/Automattic/studio/pull/523/commits/5bfc33f9dcd71909e0f514eb2dd6df6823c6379a

**Review by wojtekn — APPROVED** [review]

**Pull request merged**